assets/js/timetree.js

```diff
@@ -222,10 +233,7 @@ function TreeGraph({ nodes, links, centuries }) {
   // create a section for the background
   let background = svg.append("g").attr("id", "background");
   // visual debugging layer
-  const debugLayer = svg
-    .append("g")
-    .attr("id", "debug")
-    .attr("visibility", "hidden");
+  const debugLayer = svg.append("g").attr("id", "debug").style("opacity", 0); // not visibly by default
```

Comment:
Did pre-commit styler change? ~Is this line semantically identical?~ Oh I see, we've changed visibility hidden to opacity 0.

Reply:
right — previously I had it hard-coded and was manually changing it, but when I decided to use a slider switched to opacity